Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -0,0 +1,1350 @@
+from CommonServerPython import *
+
+''' IMPORTS '''
+from typing import Dict, Tuple, List, Union
+import urllib3
+
+"""Example for Analytics and SIEM integration
+"""
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+'''GLOBALS/PARAMS'''
+
+INTEGRATION_NAME = 'cisco-fp'
+# lowercase with `-` dividers
+INTEGRATION_COMMAND_NAME = 'ciscofp'
```

Comment:
DId I tell you to change this from `cisco-fp` to `ciscofp`?

Reply:
yes